ros_ign_gazebo/scripts/ign_gazebo

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
For some reason this file was giving me `Permission denied`. I had to do `chmod +x`